specification/attributes/discount_handling.md

```diff
@@ -0,0 +1,44 @@
+# Discount Handling
+
+A discount is a type of pricing model in which cloud providers offer their customers a reduced price that is applied to the usage of the cloud provider’s services. There are many types of discounts that a cloud provider can support, including but not limited to commercially negotiated discounts, commitment-based discounts that offer lower prices when you agree to a certain amount of their services or spend, and bundled discounts where you receive free or discounted usage of one product or service based on the usage of another product or service. Discount handling is commonly used in scenarios for verifying discounts were applied and calculating cost savings.
+
+Some of the discount offers can be purchased from a provider in order to get reduced prices. The most common example of this is commitment-based discounts, where you "purchase" a commitment to use or spend a specific amount within a period. There are instances where a portion of the purchased discount may go unused. This can result in effectively lost savings and needs to be clearly identifiable at a granular level to enable FinOps scenarios. In order to facilitate this, the amount of a purchased discount is amortized over the term that the discount is applied to (e.g., 1 year). Amortization is a process used to break down and spread the purchase costs over a period of time or term of use. When a purchase is applicable to resources, like commitment-based discounts, the amortized cost of a resource takes the initial payment and term into account and distributes it out based on the resource's usage, attributing the prorated cost for each hour of billing. Amortization provides a method for FinOps practitioners to reallocate purchase charges to the appropriate audience in support of their cost allocation efforts. Discount Handling for purchased commitments is commonly used for scenarios like calculating utilization and implementing chargeback of the purchase amount.
```

Comment:
Simplifying a tad to shorten it...
```suggestion
Some discount offers can be purchased from a provider to get reduced prices. The most common example is commitment-based discounts, where you "purchase" a commitment to use or spend a specific amount within a period. There are instances where a portion of the purchased discount may go unused, which results in lost savings and needs to be clearly identifiable at a granular level. To facilitate this, the purchase amount is amortized over the term the discount is applied to (e.g., 1 year). Amortization is a process used to break down and spread purchase costs over a period of time or term of use. When a purchase is applicable to resources, like commitment-based discounts, the amortized cost of a resource takes the initial payment and term into account and distributes it out based on the resource's usage, attributing the prorated cost for each hour of billing. Amortization enables FinOps practitioners to distribute purchase charges to the appropriate audience in support of cost allocation efforts. Discount Handling for purchased commitments is commonly used for scenarios like calculating utilization and implementing chargeback for the purchase amount.
```

Reply:
Updated my suggestion above to merge in @rodneyjoyce's comment about using "unit" instead of "hour".